packages/core/src/AvatarGroup/AvatarGroupContext.tsx

```diff
@@ -0,0 +1,33 @@
+import { createContext, useContext, useMemo } from "react";
+
+type HvAvatarGroupContextProviderProps = {
+  size: "xs" | "sm" | "md" | "lg" | "xl";
```

Comment:
We can import `HvSize` here and below (`HvAvatarGroupContextProp`) to avoid listing all sizes